app/actions/package_update.rb

```diff
@@ -9,6 +9,11 @@ def initialize
 
     def update(package, message)
       package.db.transaction do
+        if package.type == PackageModel::DOCKER_TYPE
+          package.docker_username = message.username unless message.username.nil?
+          package.docker_password = message.password unless message.password.nil?
+          package.save
+        end
         MetadataUpdate.update(package, message)
       end
       @logger.info("Finished updating metadata on package #{package.guid}")
```

Comment:
It's not only metadata anymore.

```suggestion
      @logger.info("Finished updating package #{package.guid}")
```